microsoft-edge/devtools-guide-chromium/sources/index.md

```diff
@@ -87,49 +279,292 @@ script.integrity = 'sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=';
+The Debugger includes the **Debugger** pane, along with breakpoints that are set on lines of code in the **Editor** pane.
+
+<!-- "sources-paused-breakpoint.msft.png" is not currently directly used. It's the upstream source for a couple images, including the below image. -->
+
+:::image type="complex" source="../media/sources-paused-breakpoint-highlight-debug-pane.msft.png" alt-text="The Debugger pane of the Sources tool  " lightbox="../media/sources-paused-breakpoint-highlight-debug-pane.msft.png":::
+   The **Debugger** pane of the Sources tool  
+:::image-end:::  
+
+Step through experimental changes to the code, while watching any JavaScript expressions you specify.  Watch and manually change variable values, and automatically show which variables are currently in-scope for the current line of code.
+
+The Debugger supports standard debugging actions, such as:  
+*   Setting breakpoints, to pause code.
+*   Stepping through code.
+*   Viewing and editing properties and variables.
+*   Watching the values of JavaScript expressions.
+*   Viewing the call stack (the sequence of function calls so far).
+
+The JavaScript Debugger in DevTools, in the **Debugger** pane, is designed to look and feel and work like 
+[the debugger in Visual Studio Code][CodeVisualStudioComDocsEditorDebugging] and
+[the debugger in Visual Studio][DMCVisualStudioDebuggerNavigatingThroughCodeWithTheDebugger].
+
+The following subsections cover the **Debugger** pane or breakpoints:
+*   [The basic approach to using a debugger](#the-basic-approach-to-using-a-debugger)
+*   [Advantages of the debugger's Watch and Scope over console.log](#advantages-of-the-debuggers-watch-and-scope-over-consolelog)
+*   [Articles about debugging](#articles-about-debugging)
+
+### The basic approach to using a debugger
+
+To troubleshoot where your JavaScript is going wrong, you can insert `console.log()` statements in the **Editor** pane.  Another approach, that's more powerful (yet can also be simple), is to use the debugger of Microsoft Edge DevTools.
+
+The basic approach to using a debugger is to set a breakpoint (an intentional stopping place in the code), run the code at full speed up to the line of code on which the breakpoint is set, and then step through the lines of code, starting at that breakpoint.  
+
+For example, the tutorial [Get started with debugging JavaScript][DevtoolsGuideChromiumJavascriptIndex] uses an existing, simple webpage 
+[Demo: Get Started Debugging JavaScript with Microsoft Edge (Chromium) DevTools][DevtoolsGlitchMeDebugJsGetStarted], that contains a few form controls.  
 
-By default, when you edit a file in the **Sources** tool, those changes are lost when you refresh the page.  **Workspaces** enable you to save the changes that you make in DevTools to your file system.  Essentially, DevTools is able to be used as your code editor.
+As shown in the tutorial, to get into the "Paused" mode of the debugger, do something like the following steps:
```

Comment:
Isn't this content duplicative with the one we have in the demo in the debugging doc you linked? I think it either makes more sense to direct customers to the debugging doc or to move the content from that doc here. I'm not sure copying it over again, in less depth, adds value

Reply:
I streamlined this section and repositioned it with more emphasis on the general case than on the tutorial demo in particular.

The difference between the info in this article and in the demo tutorial article is the zoom level: overview (the least you need to know) vs. detail.  The "overview" pattern is used throughout this Overview article: this article presents a compact (1-screen) summary of a given feature of the Sources tool, and then links to a detailed (multi-page) article in case the reader wants more information.